active/radiobuttons/RadioButtons_Spec.md

```diff
@@ -0,0 +1,288 @@
+# Background
+Radio buttons are used for mutually exclusive options in an app. When the users selects
+one of a group, the previously-checked button is unchecked. Consequently, radio buttons
+must be grouped, which is done with the Xaml [RadioButton](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.RadioButton)
+control by setting the GroupName property on each button to the same value.
+
+That mechanism is awkward, and doesn't by default give the right keyboarding or
+accessibility behavior.
+
+The RadioButtons control in this spec is introduced to help solve those issues.
+It's like a ListView control, but the items in the list are RadioButton controls.
+
+# Description
+
+RadioButtons is a new control that enables you to create groups of RadioButton elements easily,
+while also correctly supporting keyboarding and narrator functionality.
```

Comment:
I'm sure we don't have any ties specifically to Narrator in the implementation, so we shouldn't say so here. (Talk about accessibility in general, not Narrator in particular.)